### DISCUSSION TITLE
Endpoint to get subjects in a group

### DISCUSSION THREAD

**Discussion by mekhlakapoor** [AllenNeuralDynamics/aind-metadata-service#240] [Ideas]
**Is your feature request related to a problem? Please describe.**
As a user, I want to be able to pull metadata for all the mice in a group. Given a group ID, fetch a list of subject_ids from labtracks. Maybe then use the subject and procedure endpoints to fetch the metadata for these mice? 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by jtyoung84** [comment]
@saskiad Do you have any suggestions for this?
